main/util/src/mill/util/Jvm.scala

```diff
@@ -39,6 +39,30 @@ object Jvm extends CoursierSupport {
     os.proc(commandArgs).call(cwd = workingDir1, env = envArgs)
   }
 
+  /**
+   * A version of [[Jvm.callSubprocess]] that does not raise an exception on a non-zero exit.
```

Comment:
Can we add a `check: Boolean = true` parameter to the normal `Jvm.callSubprocess`? We can add a forwarder with the older signature to preserve binary compatibility

Reply:
How about renaming the `streamOut` parameter? It's not used in the implementation.

Reply:
Let's go with adding another parameter. Re-using existing flags is a bad idea in general; people have lost billions of dollars due to that kind of mistake haha https://blog.statsig.com/how-to-lose-half-a-billion-dollars-with-bad-feature-flags-ccebb26adeec